cpp/lazperf/io.cpp

```diff
@@ -400,6 +453,23 @@ void basic_file::Private::updateMinMax(const las::point10& p)
     header.maximum.z = (std::max)(z, header.maximum.z);
 }
 
+uint64_t basic_file::Private::chunk()
+{
+    pcompressor->done();
+    chunks.push_back( {chunk_point_num, chunk_offset });
+    pcompressor = build_las_compressor(stream->cb(), header.point_format_id, header.ebCount());
+    chunk_offset = (uint64_t)f->tellp();
```

Comment:
what happens in javascript with this cast?

Reply:
No idea. Is there some reason to think it would be bad?